debian_packages.yaml

```diff
@@ -1,6 +1,6 @@
 # debian 11
 - distros: ["debian11"]
-  archs: ["amd64", "arm64", "arm", "s390x", "ppc64le"]
+  archs: ["amd64", "arm64", "arm", "s390x", "ppc64le", "riscv64"]
```

Comment:
riscv64 should be limited to only the static images and dependencies. This will reduce impact on the build.

There's no need to pull in all packages for riscv64

Reply:
Thank you for your time!

How should I update this file?

Also I've got following error while trying `./updateWorkspaceSnapshots.sh`:
```
..URL: https://snapshot.debian.org/archive/debian/20230314T211539Z/dists/bullseye/main/binary-riscv64/Packages.xz, Status: 404 NOT FOUND
panic: failed (status: 404) to fetch remote file: "https://snapshot.debian.org/archive/debian/20230314T211539Z/dists/bullseye/main/binary-riscv64/Packages.xz"
```

So there is no snapshot for riscv64.
Is it even possible to proceed having this error?

Reply:
honestly I don't know where the riscv64 images even come from. I can't even find the non-snapshot repo (after taking a quick look). Best I could find is that the port is still "in progress" -- https://www.debian.org/ports/

Reply:
Oh, looks like I've got it now, sorry, I've updated BUILD file.

Reply:
would probably need the team that owns that port to commit to some sort of snapshotting for this to even work.